thoth/adviser/boots/solved_software_environment.py

```diff
@@ -78,7 +83,18 @@ def run(self) -> None:
         _LOGGER.warning("%s - %s", msg, self._JUSTIFICATION_LINK)
         _LOGGER.warning("Available configurations:")
 
-        configurations = self.context.graph.get_solved_python_package_versions_software_environment_all()
+        solvers = _THOTH_ADVISER_DEPLOYMENT_CONFIGURED_SOLVERS.split(" ")
+        configurations = []
+        for solver in solvers:
+            item = _OPENSHIFT.parse_python_solver_name(solver)
```

Comment:
Might be good idea to strip possible whitespace:

```suggestion
            item = OpenShift.parse_python_solver_name(solver.strip())
```

No need to instantiate OpenShift, the `parse_python_solver_name` is a class method.